### PR TITLE
feat: [CDS-78982]: onboard cd module to nav v2

### PR DESCRIPTION
**Pull request by chetanvekariya-harness**
### Summary
- Onboarded overview and get started page links in cd module and render them based on number of pipelines in project
- Onboarded Monitored Serivces and Overrides pages
- Onboarded GitOps pages
- Fix routes for Execution History

#### Screenshots
Screenshot 2023-09-12 at 12 30 15 PM

Screenshot 2023-09-12 at 2 52 07 PM
Screenshot 2023-09-12 at 2 55 08 PM

Screenshot 2023-09-12 at 4 23 13 PM

Screenshot 2023-09-12 at 2 52 20 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cd/components/CDSideNav/CDSideNavLinks.tsx [hunk 1]
```diff
@@ -51,11 +122,27 @@ const CDSideNavLinks = (mode: NAV_MODE): React.ReactElement => {
             icon="nav-home"
             label={getString('overview')}
             to={routes.toOverview({ accountId, projectIdentifier, orgIdentifier, module })}
+            visible={!isCommunity && !isCDGetStartedVisible}
           />
         </SideNav.Scope>
         <SideNav.Scope
-          scope={[Scope.PROJECT, Scope.ORGANIZATION]}
+          scope={[Scope.PROJECT]}
```
Comment (avinashmadhwani02) [inline src/modules/75-cd/components/CDSideNav/CDSideNavLinks.tsx:129]: Array is not required

**Review by avinashmadhwani02 — APPROVED** [review]

**Comment by chetanvekariya-harness** [comment]
retrigger cypress-cd

**Comment by chetanvekariya-harness** [comment]
retrigger cypress-pipeline

**Comment by chetanvekariya-harness** [comment]
retrigger cypress-cd

**Comment by chetanvekariya-harness** [comment]
retrigger cypress-pipeline

File: src/modules/75-cd/components/CDSideNav/CDSideNavLinks.tsx (hunk 1 above)
Comment (chetanvekariya-harness) [reply src/modules/75-cd/components/CDSideNav/CDSideNavLinks.tsx:129]: Made the change.

**Pull request merged**